source/nodejs/adaptivecards-templating/src/template-engine.ts

```diff
@@ -49,6 +53,23 @@ class EvaluationContext {
     set $data(value: any) {
         this._$data = value;
     }
+
+    generateVersionJson() {
+        const version = pkg.version;
+        const versionSplit = version.split('.');
+
+        let patchSplit = [];
+        if (versionSplit[2]) {
+            patchSplit = versionSplit[2].split('-');
```

Comment:
is this to account for something like `2.3.0-alpha.0`?

Reply:
Yes, it is so we can separate the patch and suffix